vegafusion-common/src/data/table.rs

```diff
@@ -267,6 +269,17 @@ impl VegaFusionTable {
         }
     }
 
+    // TODO: when updated to latest arrow version, the below function can change to
+    // pub fn from_arrow_c_stream(table: pyo3_arrow::PyTable) -> PyResult<Self> {
+    //     let (batches, schema) = table.into_inner();
+    //     Ok(VegaFusionTable::try_new(schema, batches)?)
+    // }
```

Comment:
Only a couple lines of code with pyo3_arrow!

Reply:
Aside: I never know whether there's consistency in the ecosystem between an ordering of `(schema, batches)` or `(batches, schema)` 🫠